src/services/PaymentsService.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * PaymentsService
+ * Provide services related to Payments (Nodo) to communicate with PagoPa and Backend API
+ */
+
+import { Either, Left, Right } from "fp-ts/lib/Either";
+
+import { clients as pagoPaSoapClient } from "italia-pagopa-api";
+import {
+  InodoAttivaRPTInput,
+  InodoAttivaRPTOutput,
+  InodoVerificaRPTInput,
+  InodoVerificaRPTOutput
+} from "italia-pagopa-api/dist/wsdl-lib/PagamentiTelematiciPspNodoservice/PPTPort";
+import { PagoPaConfig } from "../Configuration";
+import { ControllerError } from "../enums/ControllerError";
+
+// Send a request to PagoPa to check payment info
+export async function sendPaymentCheckRequestToPagoPa(
+  iNodoVerificaRPTInput: InodoVerificaRPTInput,
+  pagoPaConfig: PagoPaConfig
+): Promise<Either<ControllerError, InodoVerificaRPTOutput>> {
+  try {
+    const pagamentiTelematiciPSPNodoClientBase = await pagoPaSoapClient.createPagamentiTelematiciPspNodoClient(
+      {
+        endpoint: `${pagoPaConfig.HOST}:${pagoPaConfig.PORT}${
+          pagoPaConfig.SERVICES.PAYMENTS_CHECK
+        }`,
+        envelopeKey: "soapenv"
+      }
+    );
+
+    const pagamentiTelematiciPSPNodoClient = new pagoPaSoapClient.PagamentiTelematiciPspNodoAsyncClient(
+      pagamentiTelematiciPSPNodoClientBase
+    );
+
+    const nodoVerificaRPT = await pagamentiTelematiciPSPNodoClient.nodoVerificaRPT(
+      iNodoVerificaRPTInput
+    );
+
+    return new Right(nodoVerificaRPT);
```

Comment:
prefer static factories for Eithers:
`return new Right(nodoVerificaRPT); `-> `return right(nodoVerificaRPT)`